internal/command/action.go

```diff
@@ -200,3 +212,26 @@ func returnStatusOfError(err error) (int, error) {
 	}
 	return 0, err
 }
+
+//Prints a list of all provider versions
+func printProviderVersions()(error){
+	providers, err := prov.GetAllProviders()
+
+	if err != nil {
+		return err
+	}
+
+	var providerVersions bytes.Buffer
+
+	for _, provider := range providers {
+		version, err := exec.Command(path.Join(prov.DefaultPath, provider), "--version").Output()
+		if err != nil {
+			providerVersions.WriteString(provider+" version uknown")
+			continue
+		}
+		providerVersions.WriteString(provider+" version "+string(version[0:len(version)]))
```

Comment:
`string(version[0:len(version)])` I think can be done with `version.String()`? There's a construct for this for sure.

Reply:
This block with newline also can be simplified with `providerVersions.WriteString(fmt.Printf("%s version %s", provider, version))` I think

Reply:
* `string(version[0:len(version)])` was actually not needed at all anymore, because I stopped chopping off the newline.

Reply:
The newline being appended by itself was actually a bug that I missed. It should be appended to the end of the `provider version unknown` string